sycl/doc/extensions/experimental/sycl_ext_oneapi_device_architecture.asciidoc

```diff
@@ -588,6 +593,7 @@ feature, the application must be compiled in ahead-of-time (AOT) mode using
 "special target values" listed in the link:../../UsersManual.md[users manual]
 description of the `-fsycl-targets` option.  These are the target names of the
 form "intel_gpu_*", "nvidia_gpu_*", or "amd_gpu_*".
+The support of AOT compilation for `intel_cpu_spr` is currently unavailable.
```

Comment:
This should be expanded.  Create a new paragraph like:

> The architecture enumeration `intel_cpu_spr` does not currently work with any of the APIs described in this extension.  It cannot be used with the `if_architecture_is` function, the `device::ext_oneapi_architecture_is` function, or the `info::device::architecture` query descriptor.  It currently exists only for use with the sycl_ext_oneapi_matrix extension.

(It would probably good to make "sycl_ext_oneapi_matrix" a link to the spec.)